doc/arkode/guide/source/Mathematics.rst

```diff
@@ -926,6 +967,217 @@ support has been deprecated in favor of the SUNAdaptController class,
+
+Finally, by estimating the time derivative via use of finite-differences,
+
+.. math::
+   \frac{\mathrm d}{\mathrm dt} f(t_0,y_0) \approx \frac{1}{\delta t} \left(f(t_0+\delta t,y_0+\delta t f(t_0,y_0)) - f(t_0,y_0)\right),
+
+we obtain
+
+.. math::
+   |h^S_0| = \frac{(\delta t)^{1/2}}{\|f(t_0+\delta t,y_0+\delta t f(t_0,y_0)) - f(t_0,y_0)\|^{1/2}}.
+   :label: H0_TSExp1
+
+Initial step size estimation based on the simpler Taylor expansion :eq:`TSExp0`
+instead assumes that the first calculated time step should be "close" to the
+initial state,
+
+.. math::
+   &\|y(t_0+h^S_0) - y_0 \| \approx \left\|h^S_0 f(t_0,y_0)\right\| < 1,\\
+   \Rightarrow\quad&\\
+   &|h^S_0| = \frac{1}{2\left\| f(t_0,y_0)\right\|},
+   :label: H0_TSExp0
+
+where we again satisfy the inequality with a value of :math:`\frac12`.
+
+Comparing the two estimates :eq:`H0_TSExp0` and :eq:`H0_TSExp1`, we see that the
+former has double the number of :math:`f` evaluations, but that it has a less
+conservative estimate of :math:`h^S_0`, particularly since we expect any valid
+time integration method to have at least :math:`\mathcal{O}(h^S)` accuracy.
+
+Under an assumption that conservative steps will be selected for both time scales,
+the error arising from temporal coupling between the slow and fast methods may be
+negligible.  Thus, we estimate initial values of :math:`h^S_0` and :math:`h^F_0`
+independently.  Due to our assumed higher cost of :math:`f^s`, then for the slow
+time scale we employ the initial estimate :eq:`H0_TSExp0` for :math:`h^S_0` using
+:math:`f = f^s`.  Since the function :math:`f^f` is assumed to be cheaper, we
+instead apply the estimate :eq:`H0_TSExp1` for :math:`h^F_0` using :math:`f=f^f`,
+and enforce an upper bound :math:`|h^F_0| \le \frac{|h^S_0|}{10}`.
+
+.. note::
+
+   If the fast integrator does not supply its "full RHS function" :math:`f^f`
+   for the MRI method to call, then we simply initialize :math:`h^F_0 = \frac{h^S_0}{100}`.
+
+.. _ARKODE.Mathematics.MultirateFastError:
+
+Fast temporal error estimation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+MRI temporal adaptivity requires estimation of the temporal errors that
+arise at *both* the slow and fast time scales, which we denote here as
+:math:`\varepsilon^s` and :math:`\varepsilon^f`, respectively.  While the
+slow error may be estimated as :math:`\varepsilon^s = \|y_n - \tilde{y}_n\|`,
+non-intrusive approaches for estimating :math:`\varepsilon^f` are more
+challenging.  ARKODE provides two strategies to help provide this estimate, both
+of which assume that at each sub-step of the fast integrator :math:`t_{n,m}`, the
+method computes an estimate of the local temporal error, :math:`\varepsilon^f_{n,m}`
+(this typically results from its own time step solution and embedding), and that
+the fast integrator itself is temporally adaptive.  In this case, we assume that the
+fast integrator was run with the same absolute tolerances as the slow integrator, but
```

Comment:
I don't think it's wise for an adaptivity strategy to scale absolute tolerances, since for users who follow our guidance, ATOL should be the floor, below which anything is considered to be "noise".  I suppose that for applications where the relative tolerance is useless (i.e., zero-valued solutions are expected) then this entire MRI adaptivity approach may not really make sense, in which case perhaps we should recommend for users to use an alternate strategy?